pykickstart/commands/ostreecontainer.py

```diff
@@ -96,3 +96,8 @@ def parse(self, args):
 
 class RHEL9_OSTreeContainer(F38_OSTreeContainer):
     pass
+
+class F43_OSTreeContainer(F38_OSTreeContainer):
+    removedKeywords = KickstartCommand.removedKeywords
+    removedAttrs = KickstartCommand.removedAttrs
+    conflictingCommands = ["ostreesetup", "bootc"]
```

Comment:
Please add bootc to the ostreecontainer conflicting test

Reply:
Done